DataDog/documentation
# add source comment to workflow automation .md

### What does this PR do? What is the motivation?


adds comment to workflow/action_catalog .md files (pulled files) at build time that indicates what external repo it was sourced from.

### Preview
there should be no ui change. check that no action pages have changed:

https://docs-staging.datadoghq.com/stefon.simmons/workflows-source-comment/service_management/workflows/actions_catalog/

### Merge instructions


- [x] Please merge after CorpWeb team has reviewed

### Additional notes

to test: 
- clone repo locally
- checkout this branch (`stefon.simmons/workflows-source-comment`)
- run `make start`
- check that `content/en/service_management/workflows/actions_catalog` files have a comment above the markdown under the frontmatter.
  - ex: ``
